### ISSUE TITLE
Notice on imagecolorat()

### ISSUE THREAD

**Issue by geri777** [verot/class.upload.php#162]
Notice: imagecolorat(): 45,2848 is out of bounds in class.upload.php on line 4902
The image has a height of 2848 pixels!

```
original:
$rgba = imagecolorat($image_dst, $x, $y);

fixed:
$rgba = imagecolorat($image_dst, $x, $y == 0 ? 0 : $y - 1);
```

The size seems to be a null-based index.
https://stackoverflow.com/questions/38340662/php-get-color-at-last-pixels-of-image-error

**Comment by verot** [comment]
Thank you for reporting the issue. Can you provide me with a test case?

Besides, is the problem affecting all calls to `imagecolorat()` or only this one call?

**Comment by verot** [comment]
Should be fixed in c618591
